src/store/store.js

```diff
@@ -0,0 +1,16 @@
+import create from 'zustand';
```

Comment:
default import 방식은 deprecated 됐다고 합니다.

```suggestion
import { create } from 'zustand';
```
위처럼 수정 부탁드려요!